test/replication/long_row_timeout.test.lua

```diff
@@ -8,7 +8,7 @@ test_run = require('test_run').new()
 box.schema.user.grant('guest', 'replication')
 test_run:cmd('create server replica with rpl_master=default, script="replication/replica.lua"')
 test_run:cmd('start server replica')
-box.info.replication[2].downstream.status
+test_run:wait_downstream(2, {status = 'follow'})
```

Comment:
1. `test_run:cmd('start server replica')` means waiting for full database initialization. In case of replica it means full recovery from master. So, if you see something that is not 'follow' after this, it means that the replica fails to bootstrap.
2. Your output in the commit message shows 'stopped' status. AFAIU, it is the terminal state. So, if you're here, you'll not get 'follow' by waiting.

Your description:

> It happened because replication downstream status check occurred too
> early, when it was only in 'stopped' state.

Does it mean that the 'start server' command does not work as expected?

Reply:
Hm. Just got that we acquire the status not on the replica, but on its master. So it looks logical that the status will not be updated instantly. However there still question, how we observed the 'stopped' status. Whether it is some side effect of a previous test? Whether the observed behaviour is the same after https://github.com/tarantool/test-run/pull/309?

Reply:
Observed behavior is the same after added patch for server restart. Actually it's not the first time when on slow hosts like VMware with FreeBSD it shows 'stopped' status and than changes to 'follow'.

Reply:
Found that this situation happens when replica done its initial join, but not reached subscription yet.